tests/sklearn/test_sklearn_models.py

```diff
@@ -1871,3 +1946,29 @@ def test_rounding_consistency_for_regular_models(
         metric,
         is_weekly_option,
     )
+
+
+@pytest.mark.parametrize("model_class, parameters", get_sklearn_tree_models_and_datasets())
+@pytest.mark.parametrize("n_bits", [2, 5, 10])
+def test_fhe_sum_for_tree_based_models(
+    model_class,
+    parameters,
+    n_bits,
+    load_data,
+    is_weekly_option,
+    verbose=True,
+):
+    """Test that Concrete ML without and with rounding are 'equivalent'."""
+
+    if verbose:
+        print("Run check_fhe_sum_consistency")
+
+    x, y = get_dataset(model_class, parameters, n_bits, load_data, is_weekly_option)
+
+    check_fhe_sum_consistency(
```

Comment:
better to keep the same name between check and tests : `check_fhe_sum_for_tree_based_models` !